RationaleMCP/ReadMe.md

```diff
@@ -24,10 +24,10 @@ but the rest of the development on a branch/pull-request before being accepted.
 |Active|0038|Initialization of Clocked Partitions|[MCP/0038](https://github.com/modelica/ModelicaSpecification/tree/MCP/0038/RationaleMCP/0038)||
 |Active|0037|Generalized Modelica URIs|[MCP/0037](https://github.com/modelica/ModelicaSpecification/tree/MCP/0037/RationaleMCP/0037)|https://github.com/modelica/ModelicaSpecification/pull/2663|
 |Active|0036|Setting states|[MCP/0036](https://github.com/modelica/ModelicaSpecification/tree/MCP/0036/RationaleMCP/0036)|https://github.com/modelica/ModelicaSpecification/pull/3164|
-|Active|0035|Multilingual support of Modelica|[MCP/0035](https://github.com/modelica/ModelicaSpecification/tree/MCP/0035/RationaleMCP/0035)|https://github.com/modelica/ModelicaSpecification/pull/2956|
+|Added in [3.6](https://github.com/modelica/ModelicaSpecification/releases/tag/v3.6)|0035|Multilingual support of Modelica|[MCP/0035](https://github.com/modelica/ModelicaSpecification/tree/MCP/0035/RationaleMCP/0035)|https://github.com/modelica/ModelicaSpecification/pull/2956|
```

Comment:
We can't merge something saying it has been added to 3.6 before 3.6 has been released.  If the intention is that some parts of this PR should be merged before the release, the PR needs to be split.

Reply:
Alternatively, we begin by just changing the status to _Accepted_.

Reply:
Ok, doing that. The idea was to have one self-contained pull request. There are similar problems with the revisions where the phrase "will be voted on" is used.

Reply:
Resolving according to yesterday's phone meeting decision to use _master_ as the release preparation branch.